### DISCUSSION TITLE
Security onion rules management

### DISCUSSION THREAD

**Discussion by notepas** [Security-Onion-Solutions/securityonion#5711] [General] [answered]
Hello,

Is it possible to modify existing alert rules in Security Onion through any graphical editor?
I find a way how to do disable and enable rule through the console, but I can't find how to modify existing rule through the graphical interface.

**Comment by clairmont32** [comment]
This currently is not a feature of SO. I havent seen an open source method of managing Suricata rules through GUI but that doesnt mean one may not exist somewhere.

**Comment by notepas** [comment]
Thank you.

**Comment by weslambert** [comment, marked as answer]
This is a feature we plan on having in the future as we continue to work on increased management of the grid via a web interface.
